kinto/__main__.py

```diff
@@ -31,10 +31,23 @@ def main(args=None):
                         required=False,
                         default=DEFAULT_CONFIG_FILE)
 
-    parser.add_argument('-v', '--version',
+    parser.add_argument('--version',
                         action='version', version=__version__,
                         help='Print the Kinto version and exit.')
 
+    # Defaults
+    parser.add_argument('-v', '--verbose', action='store_const',
+                        const=logging.INFO, dest='verbosity',
+                        help='Show all messages.')
+
+    parser.add_argument('-q', '--quiet', action='store_const',
+                        const=logging.CRITICAL, dest='verbosity',
+                        help='Show only critical errors.')
+
+    parser.add_argument('-D', '--debug', action='store_const',
```

Comment:
Maybe a matter of taste but I think `--debug`` is enough. `-D` is not common right?